pallets/common/src/traits/relayer.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{traits::identity, CommonConfig};
+use frame_support::dispatch::DispatchResult;
+use frame_support::{decl_event, weights::Weight};
+use polymesh_primitives::{EventDid, IdentityId, Signatory};
+
+pub trait WeightInfo {
+    fn set_paying_key() -> Weight;
+    fn accept_paying_key() -> Weight;
+    fn remove_paying_key() -> Weight;
+    fn update_polyx_limit() -> Weight;
+}
+
+/// This trait is used to add a paying key to a user key.
+pub trait IdentityToRelayer<Balance, AccountId> {
+    /// Accepts and adds a paying key to a user key
+    ///
+    /// # Arguments
+    /// * `signer` - DID/key of the signer.
+    /// * `from` - The DID that sent the authorization.
+    /// * `user_key` - The user key.
+    /// * `paying_key` - The paying key.
+    /// * `polyx_limit` - Initial Polyx limit.
+    fn auth_accept_paying_key(
+        signer: Signatory<AccountId>,
+        from: IdentityId,
+        user_key: AccountId,
+        paying_key: AccountId,
+        polyx_limit: Balance,
+    ) -> DispatchResult;
+}
```

Comment:
Let's rebase atop of my https://github.com/PolymathNetwork/Polymesh/pull/1035 PR and nix this.

Reply:
(merged now)